heareval/score.py

```diff
@@ -53,6 +54,28 @@ def label_to_binary_vector(label: List, num_labels: int) -> torch.Tensor:
     return binary_labels
 
 
```

Comment:
So I've seen us do this a few times, and I am not sure I love this pattern. That we have simple types and then, changing to more complex types, we *overload* the existing function and then use `isinstance` for runtime typing. It seems more fragile than breaking a function into something with a very specific return type, which we can statically check using `mypy`.

Do you think we can fix this easily? If not, let's make an issue to revisit this?

Reply:
I guess this is okay, since it's only two types if we remove the ints.